Scyther Playground/Scyther Playground/Button.swift

```diff
@@ -187,23 +192,25 @@ class Button: UIButton {
         if type == .plainLeft {
             contentHorizontalAlignment = .left
         }
-
+        
         //Set Constants
-        titleLabel?.font = type.titleFont
+        titleLabel.font = type.titleFont
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
         layer.cornerRadius = type.cornerRadius
         clipsToBounds = true
         backgroundColor = type.backgroundColor
-        imageView?.tintColor = type.iconTintColor
+        imageView.tintColor = type.iconTintColor
         layer.borderColor = type.borderColor
         layer.borderWidth = type.borderWidth
         setTitleColor(type.titleColor, for: .normal)
         setTitleColor(type.disabledBackgroundColor, for: .disabled)
         
         //Set Constraints
-        titleLabel?.snp.remakeConstraints({ (make) in
-            make.height.equalTo(type.buttonHeight)
-            make.top.bottom.equalToSuperview()
-        })
+        NSLayoutConstraint.activate([
+            titleLabel.heightAnchor.constraint(equalToConstant: type.buttonHeight),
+            titleLabel.topAnchor.constraint(equalTo: superview.topAnchor),
```

Comment:
Why is there now a new top anchor here?

Reply:
the old code had a hight constraint and a top and bottom constraint:
```
titleLabel?.snp.remakeConstraints({ (make) in
            make.height.equalTo(type.buttonHeight)
            make.top.bottom.equalToSuperview()
        })
```